kickstart/provision.erb

```diff
@@ -183,12 +183,14 @@ echo "updating system time"
 echo 'proxy = <%= proxy_uri %>' >> /etc/yum.conf
 <% end -%>
 
+<% unless @host.param_false?('update_base_packages') -%>
```

Comment:
It used in the cloudinit template to run `yum upgrade` (for EL-based distros anyway, cloud-init abstracts it) automatically. I don't think the minor distinction between update and upgrade is important or particularly intentional, they both upgrade the majority of packages (not just refresh metadata).

The parameter should probably be package_upgrade.